base/pkg/entry.jl

```diff
@@ -50,19 +50,20 @@ function add(pkg::AbstractString, vers::VersionSet)
     outdated = :maybe
     @sync begin
         @async if !edit(Reqs.add,pkg,vers)
-            ispath(pkg) || throw(PkgError("unknown package $pkg"))
```

Comment:
It's a top level function (`Pkg.add` just wrapper around it) that serves as Pkg UI. So why do we need to dump a stack trace just to inform user that package does not exist?